### ISSUE TITLE
Newly joined groups have no channels

### ISSUE THREAD

**Issue by mursicale** [urbit/urbit#3669]
**Describe the bug**
Some groups after joining do not show any channels belonging to the group, and in the navigation sidebar the group shows up with `ship/<@p of host>/<name of group>`, rather than just `<name of group>`. The specific groups affected for me have been

- ~noltev-dabrud/---computer-music 
- ~sorwet/veep-bleachers
- ~wicdev-wisryt/debate

Other than groups that I had to re-join because of the previous issue with groups after an OTA, these are all the most recent groups that I have joined.
Most of them were created after that OTA, but I believe that the computer-music group may have been created before.

**To Reproduce**
Join a group, and ... this just happens

**Expected behavior**
Expected to see chats, journals, links, and such.

**Desktop (please complete the following information):**

 - Linux bunko 5.7.0-1-amd64 #1 SMP Debian 5.7.6-1 (2020-06-24) x86_64 GNU/Linux
 - Firefox 78.3.0esr (64-bit)
 - %base-hash 0vl.9h0dn.598eu.2q9rt.qnesu.r0no0.beuoc.fop5s.ugpa5.lage8.84sem

**Additional context**

~nartes-fasrum said they were having a similar issue with at least one group

~hastuc-dibtux recommended I include the following: 
I did remove my dms from this but I suspect that has nothing to do with the situation.

`:metadata-hook +dbug` output:
```
>   [ %1
    synced
  { [p=/ship/~nibset-napwyn/remote-life q=~nibset-napwyn]
    [p=/ship/~libhut-samwes/crypto-forum-redivivus q=~libhut-samwes]
    [p=/ship/~sorwet/veep-bleachers q=~sorwet]
    [p=/ship/~noltev-dabrud/---computer-music q=~noltev-dabrud]
    [p=/ship/~minder-folden/---the-sanctorium q=~minder-folden]
    [p=/ship/~satsyd-finsun/dm--libhut-samwes q=~satsyd-finsun]
    [p=/ship/~minder-folden/purgatory q=~minder-folden]
    [p=/ship/~hiddev-dannut/new-hooniverse q=~hiddev-dannut]
    [p=/ship/~salmus-master/the-tribal-network q=~salmus-master]
    [p=/ship/~rovnys-ricfer/arcana-imperii q=~rovnys-ricfer]
    [p=/ship/~bitbet-bolbel/urbit-community q=~bitbet-bolbel]
    [p=/ship/~littel-wolfur/----the-forge q=~littel-wolfur]
    [p=/ship/~darlur/bookends q=~darlur]
    [p=/ship/~littel-wolfur/dotfiles q=~littel-wolfur]
    [p=/ship/~mister-todteg/martian-firearms-club-7666 q=~mister-todteg]
    [p=/ship/~dasfeb/smol-computers q=~dasfeb]
    [p=/ship/~satsyd-finsun/areopagus q=~satsyd-finsun]
  }
]
```
output from `metadata-hook +dbug %bowl
```
>   [ [our=~satsyd-finsun src=~satsyd-finsun dap=%metadata-hook]
  [   wex
    { [   p
        [ wire=/group/ship/~salmus-master/the-tribal-network
          ship=~salmus-master
          term=%metadata-hook
        ]
        q=[acked=%.y path=/group/ship/~salmus-master/the-tribal-network]
      ]
      [   p
        [ wire=/group/ship/~mister-todteg/martian-firearms-club-7666
          ship=~mister-todteg
          term=%metadata-hook
        ]
        q=[acked=%.y path=/group/ship/~mister-todteg/martian-firearms-club-7666]
      ]
      [ p=[wire=/group/ship/~dasfeb/smol-computers ship=~dasfeb term=%metadata-hook]
        q=[acked=%.y path=/group/ship/~dasfeb/smol-computers]
      ]
      [ p=[wire=/group/ship/~darlur/bookends ship=~darlur term=%metadata-hook]
        q=[acked=%.y path=/group/ship/~darlur/bookends]
      ]
      [   p
        [ wire=/group/ship/~noltev-dabrud/---computer-music
          ship=~noltev-dabrud
          term=%metadata-hook
        ]
        q=[acked=%.y path=/group/ship/~noltev-dabrud/---computer-music]
      ]
      [   p
        [wire=/group/ship/~bitbet-bolbel/urbit-community ship=~bitbet-bolbel term=%metadata-hook]
        q=[acked=%.y path=/group/ship/~bitbet-bolbel/urbit-community]
      ]
      ]
      [ p=[wire=/group/ship/~minder-folden/purgatory ship=~minder-folden term=%metadata-hook]
        q=[acked=%.y path=/group/ship/~minder-folden/purgatory]
      ]
      [   p
        [ wire=/group/ship/~minder-folden/---the-sanctorium
          ship=~minder-folden
          term=%metadata-hook
        ]
        q=[acked=%.y path=/group/ship/~minder-folden/---the-sanctorium]
      ]
      [   p
        [wire=/group/ship/~hiddev-dannut/new-hooniverse ship=~hiddev-dannut term=%metadata-hook]
        q=[acked=%.y path=/group/ship/~hiddev-dannut/new-hooniverse]
      ]
      [   p
        [ wire=/group/ship/~libhut-samwes/crypto-forum-redivivus
          ship=~libhut-samwes
          term=%metadata-hook
        ]
        q=[acked=%.y path=/group/ship/~libhut-samwes/crypto-forum-redivivus]
      ]
      [   p
        [wire=/group/ship/~rovnys-ricfer/arcana-imperii ship=~rovnys-ricfer term=%metadata-hook]
        q=[acked=%.y path=/group/ship/~rovnys-ricfer/arcana-imperii]
      ]
      [ p=[wire=/group/ship/~nibset-napwyn/remote-life ship=~nibset-napwyn term=%metadata-hook]
        q=[acked=%.y path=/group/ship/~nibset-napwyn/remote-life]
      ]
      [ p=[wire=/group/ship/~littel-wolfur/dotfiles ship=~littel-wolfur term=%metadata-hook]
        q=[acked=%.y path=/group/ship/~littel-wolfur/dotfiles]
      ]
      [ p=[wire=/group/ship/~littel-wolfur/----the-forge ship=~littel-wolfur term=%metadata-hook]
        q=[acked=%.y path=/group/ship/~littel-wolfur/----the-forge]
      ]
      [p=[wire=/updates ship=~satsyd-finsun term=%metadata-store] q=[acked=%.y path=/updates]]
      [ p=[wire=/group/ship/~sorwet/veep-bleachers ship=~sorwet term=%metadata-hook]
        q=[acked=%.y path=/group/ship/~sorwet/veep-bleachers]
      ]
    }
      sup
    { [   p
        ~[
          /g/sys/req/~sichec-tonleb-locped-hacbyn--dapler-pagbex-nilned-marzod/metadata-hook
          /a/bone/~sichec-tonleb-locped-hacbyn--dapler-pagbex-nilned-marzod/29
          //ames
        ]
          q
        [ p=~sichec-tonleb-locped-hacbyn--dapler-pagbex-nilned-marzod
          q=/group/ship/~satsyd-finsun/areopagus
        ]
      ]
      [ p=~[/g/sys/req/~fonnyx-nopmer/metadata-hook /a/bone/~fonnyx-nopmer/9 //ames]
        q=[p=~fonnyx-nopmer q=/group/ship/~satsyd-finsun/dm--fonnyx-nopmer]
      ]
      [ p=~[/g/sys/req/~simfur-ritwed/metadata-hook /a/bone/~simfur-ritwed/33 //ames]
        q=[p=~simfur-ritwed q=/group/ship/~satsyd-finsun/areopagus]
      ]
      [ p=~[/g/sys/req/~radbur-sivmus/metadata-hook /a/bone/~radbur-sivmus/57 //ames]
        q=[p=~radbur-sivmus q=/group/ship/~satsyd-finsun/areopagus]
      ]
      [ p=~[/g/sys/req/~wicdev-wisryt/metadata-hook /a/bone/~wicdev-wisryt/49 //ames]
        q=[p=~wicdev-wisryt q=/group/ship/~satsyd-finsun/areopagus]
      ]
      [ p=~[/g/sys/req/~todnes-rolhul/metadata-hook /a/bone/~todnes-rolhul/13 //ames]
        q=[p=~todnes-rolhul q=/group/ship/~satsyd-finsun/areopagus]
      ]
      [ p=~[/g/sys/req/~macpur-salpyx/metadata-hook /a/bone/~macpur-salpyx/29 //ames]
        q=[p=~macpur-salpyx q=/group/ship/~satsyd-finsun/areopagus]
      ]
    }
  ]
  act=23.491
    eny
  \/0v2nv.0ae6m.me49k.e4jt4.9rrvf.ae0cm.1krvt.s0vig.drifb.ed0ec.k2dkl.98l8i.gpf55.1h9fv.p5202.s\/
    69k0.23se1.hr7fu.606jv.9p576.27pcu
  \/                                                                                           \/
  now=~2020.10.8..01.19.43..fd38
  byk=[p=~satsyd-finsun q=%home r=[%da p=~2020.9.5..09.02.20..67aa]]
]
```
**Update**

This appears to be happening every time now.
Here's what outputs when I join a group: 

```
error in %metadata-hook
%arvo-response
! /lib/default-agent/hoon:<[63 3].[63 5]>
! 
  \/"unexpected system response %g to %metadata-hook on wire /out/~rivpyl-sidfyl/metadata-hook/\/
    group/ship/~rivpyl-sidfyl/martian-cinema"
  \/                                                                                           \/
! /lib/default-agent/hoon:<[62 3].[63 5]>
error in %metadata-hook
%arvo-response
! /lib/default-agent/hoon:<[63 3].[63 5]>
! 
  \/"unexpected system response %g to %metadata-hook on wire /out/~rivpyl-sidfyl/metadata-hook/\/
    group/ship/~rivpyl-sidfyl/martian-cinema"
  \/                                                                                           \/
! /lib/default-agent/hoon:<[62 3].[63 5]>
error in %metadata-hook
%arvo-response
! /lib/default-agent/hoon:<[63 3].[63 5]>
! 
  \/"unexpected system response %g to %metadata-hook on wire /out/~rivpyl-sidfyl/metadata-hook/\/
    group/ship/~rivpyl-sidfyl/martian-cinema"
  \/                                                                                           \/
! /lib/default-agent/hoon:<[62 3].[63 5]>
```

**Comment by matildepark** [comment]
I've also heard this happening infrequently with `~bollug-worlus/urbit-index` and Hooniversity (#3641) but we don't know the exact reproduction cases.

@liam-fitzgerald @tacryt-socryp Does smell like metadata-store issues, do we know of cases where these would now fail?

**Comment by liam-fitzgerald** [comment]
Hi @mursicale can you post the output from `+trouble` as well?

**Comment by mursicale** [comment]
yeparino:
```
[%base-hash 0vl.9h0dn.598eu.2q9rt.qnesu.r0no0.beuoc.fop5s.ugpa5.lage8.84sem]
  [%home-hash 0v1l.067hv.6ao3q.sifm9.45g3r.scb6e.n5clk.6rgln.ejgpd.ckb0c.htajd]
  [%kids-hash 0v1c.o253s.lpvsm.t4fr4.en39p.mq1rj.qjnrk.tjnrl.c5t1i.o68th.naoqh]
  [%glob-hash 0v3.u1ets.ipgbo.eo23m.md70h.djpj0 %done]
  [%our ship=~satsyd-finsun point='981437487' life=[~ 2] rift=[~ 1]]
  [%sponsor ship=~nibfeb point='35887' life=[~ 4] rift=[~ 3]]
  [%dopzod ship=~dopzod point='4608' life=[~ 3] rift=[~ 2]]
```

**Comment by liam-fitzgerald** [comment]
Ok lovely, it looks like your ship didn't hasn't properly updated in a while. Can you run `|merge %home ~nibfeb %kids, =gem %that` and post the result?

**Comment by mursicale** [comment]
```
merged with strategy %that
zuse: ~disfed-fotnes
vane: %a ~maglet-masfet
vane: %b ~nocsed-radtul
vane: %c ~liblep-nosnyd
vane: %d ~havser-tagmug
vane: %e ~sitwyd-divrel
vane: %g ~tiddyn-rovfed
vane: %i ~rivdyn-fosfer
vane: %j ~wanret-figfun
gall: molting
gall: upgrading %contact-hook
gall: upgrading %dojo
gall: upgrading %chat-hook
crud: %drip-fail event failed
drip failed
[%poke p=%wake]
%leap
[%feck p=%veer]
%leap
[%feck p=%veer]
%leap
[%feck p=%veer]
%leap
[%feck p=%veer]
%leap
[%feck p=%veer]
%leap
[%feck p=%veer]
%leap
[%feck p=%veer]
%leap
[%feck p=%veer]
%leap
[%feck p=%veer]
%leap
  wire
/ use
  publish
  ~satsyd-finsun
  read
  info
  app
  publish
  notebooks
  pre-publication
  publish-info
  bar-stack
~[
  ~[/d //term/1]
  ~[/g/use/hood/~satsyd-finsun/kiln/home /d //term/1]
  ~[/g/use/hood/~satsyd-finsun/kiln/home /d //term/1]
  ~[
    /c/merge/home/~nibfeb/kids/that
    /g/use/hood/~satsyd-finsun/kiln/home
    /d
    //term/1
  ]
  ~[//behn]
]
take: failed
```
and then many similar errors to the earlier ones from default-agent saying unexpected response.

**Comment by matildepark** [comment]
Ah the chat-hook on-peek strikes again. @tacryt-socryp

**Comment by matildepark** [comment]
The root issue of the OTA crash here is fixed as per #3713, a fresh `|ota ~nibfeb %kids` should fix this.
